pkg/webhook/pod/pods.go

```diff
@@ -272,18 +272,17 @@ func (pc *PodAdmissionControl) AdmitCreatePods(ar admission.AdmissionReview) *ad
 	return util.ARSuccess()
 }
 
-func fetchInfoFromPayload(payload *admitPayload) (pod *core.Pod, ownerStatefulSet *apps.StatefulSet, ordinal int32, podName, namespace, tcName string, isInOrdinal, isUpgrading, IsDeferDeleting bool, pdClient pdapi.PDClient, err error) {
+func fetchInfoFromPayload(payload *admitPayload) (ordinal int32, podName, namespace, tcName string, isInOrdinal, isUpgrading, IsDeferDeleting bool, err error) {
```

Comment:
Still so many return values, the `podName, namespace, tcName` can be retrieved easily and the rest values are calculated only in one line, is it better to just call the target function in the caller?

Reply:
updated.